streamingvisitors/src/vespa/searchvisitor/search_environment_snapshot.h

```diff
@@ -16,13 +16,15 @@ class SearchEnvironmentSnapshot
     std::shared_ptr<const RankManager::Snapshot> _rank_manager_snapshot;
     std::shared_ptr<VsmfieldsConfig>             _vsm_fields_cfg;
     std::shared_ptr<const vsm::DocsumTools>      _docsum_tools;
+    int64_t                                      _generation;
```

Comment:
Consider rename to `_config_generation`.

Reply:
Renamed.